packages/next/src/build/output/log.ts

```diff
@@ -35,8 +35,9 @@ function prefixedLog(prefixType: keyof typeof prefixes, ...message: any[]) {
   }
 }
 
-export function bootstrap(...message: any[]) {
-  console.log(' ', ...message)
+export function bootstrap(...message: string[]) {
+  // Add space after the prefix
+  console.log('  ' + message.join(' '))
```

Comment:
> // Add space after the prefix

Not following this comment. Can you rephrase? It doesn't explain a why so I feel like the comment will be redundant.

Reply:
It's for the indentation with other logs, I added some example in the comments.

And the test plan is updated in the PR description